test/featuregates_linter/test_data/unversioned_feature_list.yaml

```diff
@@ -574,12 +574,6 @@
     lockToDefault: true
     preRelease: GA
     version: ""
-- name: PodHostIPs
-  versionedSpecs:
-  - default: true
-    lockToDefault: true
-    preRelease: GA
-    version: ""
```

Comment:
After this is migrated to the versioned feature list in https://github.com/kubernetes/kubernetes/pull/126791/files#diff-530f68e01a51f788bd0ad558bde87a4a378bd3e40e3811d097029f6aa109c7a7R272-R276, I think we need to rebase on that.


```
	PodHostIPs: {
		{Version: version.MustParse("1.28"), Default: false, PreRelease: featuregate.Alpha},
		{Version: version.MustParse("1.29"), Default: true, PreRelease: featuregate.Beta},
		{Version: version.MustParse("1.30"), Default: true, PreRelease: featuregate.GA, LockToDefault: true}, 
		{Version: mustParseVersion("1.32"), Default: false, PreRelease: Removed},

	},
```